### ISSUE TITLE
The result is always zero and the L_out is always 'nan'.

### ISSUE THREAD

**Issue by Jaychouxq** [QingyongHu/RandLA-Net#219]
****EPOCH 0****
Step 00000050 L_out=  nan Acc=0.00 --- 1408.79 ms/batch
Step 00000100 L_out=  nan Acc=0.00 --- 1448.62 ms/batch
Step 00000150 L_out=  nan Acc=0.00 --- 1471.52 ms/batch
Step 00000200 L_out=  nan Acc=0.00 --- 1353.99 ms/batch
Step 00000250 L_out=  nan Acc=0.00 --- 1416.76 ms/batch
Step 00000300 L_out=  nan Acc=0.00 --- 1384.90 ms/batch
Step 00000350 L_out=  nan Acc=0.00 --- 1479.48 ms/batch
Step 00000400 L_out=  nan Acc=0.00 --- 1401.82 ms/batch
Step 00000450 L_out=  nan Acc=0.00 --- 1405.77 ms/batch
Step 00000500 L_out=  nan Acc=0.00 --- 1225.64 ms/batch
eval accuracy: 0.18104404296875
mean IOU:0.013926464843750002
Mean IoU = 1.4%
--------------------------------------------------------------------------------------
 1.39 | 18.10  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00 
--------------------------------------------------------------------------------------

Best m_IoU is: 1.393
****EPOCH 1****
Step 00000550 L_out=  nan Acc=0.00 --- 1385.89 ms/batch
Step 00000600 L_out=  nan Acc=0.00 --- 1200.71 ms/batch
Step 00000650 L_out=  nan Acc=0.00 --- 1377.93 ms/batch
Step 00000700 L_out=  nan Acc=0.00 --- 1414.77 ms/batch
Step 00000750 L_out=  nan Acc=0.00 --- 1565.10 ms/batch
Step 00000800 L_out=  nan Acc=0.00 --- 1391.87 ms/batch
Step 00000850 L_out=  nan Acc=0.00 --- 1308.24 ms/batch
Step 00000900 L_out=  nan Acc=0.00 --- 1403.82 ms/batch
Step 00000950 L_out=  nan Acc=0.00 --- 1389.88 ms/batch
Step 00001000 L_out=  nan Acc=0.00 --- 1248.54 ms/batch
eval accuracy: 0.18180821533203126
mean IOU:0.013985247333233173
Mean IoU = 1.4%
--------------------------------------------------------------------------------------
 1.40 | 18.18  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00 
--------------------------------------------------------------------------------------

Best m_IoU is: 1.399
****EPOCH 2****
Step 00001050 L_out=  nan Acc=0.00 --- 1435.68 ms/batch
Step 00001100 L_out=  nan Acc=0.00 --- 1194.74 ms/batch
Step 00001150 L_out=  nan Acc=0.00 --- 1860.80 ms/batch
Step 00001200 L_out=  nan Acc=0.00 --- 1391.87 ms/batch
Step 00001250 L_out=  nan Acc=0.00 --- 1437.67 ms/batch
Step 00001300 L_out=  nan Acc=0.00 --- 1389.88 ms/batch
Step 00001350 L_out=  nan Acc=0.00 --- 1390.87 ms/batch
Step 00001400 L_out=  nan Acc=0.00 --- 1376.94 ms/batch
Step 00001450 L_out=  nan Acc=0.00 --- 1390.87 ms/batch
Step 00001500 L_out=  nan Acc=0.00 --- 1287.37 ms/batch

**Comment by loki-veera** [comment]
hi did you change your loss function?. looks like your loss is nan.

**Comment by Jaychouxq** [comment]
> hi did you change your loss function?. looks like your loss is nan.

I don't how to change my loss function, Why does the source code released by the author need to be modified?

**Comment by loki-veera** [comment]
The information you provided is very less to comment on so I thought you changed your loss function like changing cross entropy weights. What dataset you training on?. Also for the cross entropy loss to be NaN the input to loss should be zero.

**Comment by Jaychouxq** [comment]
> 您提供的信息很少评论，所以我认为您更改了损失函数，例如更改交叉熵权重。您在什么数据集上进行训练？此外，对于交叉熵损失为NaN，损失的输入应为零。

I didn't modify the author's code. The dataset I used was S3DIS.

**Comment by Jaychouxq** [comment]
@QingyongHu Hello, do you know why this problem arises.

**Comment by Jaychouxq** [comment]
I still don't know the cause of this problem, so I switched to Linux and succeeded
